src/components/tl/cuda/tl_cuda.h

```diff
@@ -32,7 +32,7 @@
 #define UCC_TL_CUDA_SUPPORTED_COLLS                                            \
     (UCC_COLL_TYPE_ALLTOALL | UCC_COLL_TYPE_ALLTOALLV |                        \
      UCC_COLL_TYPE_ALLGATHER | UCC_COLL_TYPE_ALLGATHERV |                      \
-     UCC_COLL_TYPE_BCAST |                                                     \
+     UCC_COLL_TYPE_BCAST | UCC_COLL_TYPE_ALLREDUCE |                           \
```

Comment:
here we need ifdef as well, otherwise we will see TL CUDA allreduce in summary table when UCC_LOG_LEVEL=info is set